content/it/_index.html

```diff
@@ -1,70 +1,63 @@
 ---
-title: Panoramica sull'amministrazione del cluster
-content_template: templates/concept
-weight: 10
+title: "Orchestrazione di Container in produzione"
```

Comment:
nit: no need for quotes
```suggestion
title: Orchestrazione di Container in produzione
```